src/components/Property/PropertyDetails.js

```diff
@@ -2,52 +2,61 @@ import PropTypes from 'prop-types'
 import LocationAlerts from './LocationAlerts'
 import PersonAlerts from './PersonAlerts'
 import Tenure from './Tenure'
+import RaiseRepairLink from './RaiseRepairLink'
+import RepairUnavailableText from './RepairUnavailableText'
 
 const PropertyDetails = ({
   address,
   hierarchyType,
   locationAlerts,
   personAlerts,
   tenure,
-}) => (
-  <div>
-    <h1 className="govuk-heading-l">
-      {hierarchyType.subTypeDescription}: {address.addressLine}
-    </h1>
-
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-one-half">
-        <div className="govuk-body-s">
-          <div className="property-details-main-section">
-            <span className="govuk-body-xs">Property details</span>
-            <br></br>
-            <span className="govuk-!-font-weight-bold text-green">
-              {address.addressLine}
-            </span>
-            <br></br>
-            {address.streetSuffix && (
-              <>
-                <span className="govuk-!-font-weight-bold text-green">
-                  {address.streetSuffix}
-                </span>
-                <br></br>
-              </>
-            )}
-            <span className="govuk-body-xs text-green">
-              {address.postalCode}
-            </span>
+}) => {
+  return (
+    <div>
+      <h1 className="govuk-heading-l repair-laber">
+        {hierarchyType.subTypeDescription}: {address.addressLine}
+      </h1>
+      <div>
+        {tenure.canRaiseRepair ? (
+          <RaiseRepairLink description={hierarchyType.subTypeDescription} />
+        ) : (
+          <RepairUnavailableText />
+        )}
```

Comment:
I'd have this like `<RaiseRepairStatus>` and have all this logic in that component. No need for two components here I don't think. Especially as the `<RepairUnavailableText />` holds no state